### PR TITLE
add share and update custom extracts in docs

### PR DESCRIPTION
**Pull request by kkowalsky**
@rmglennon take a look, added information about how to share and update custom extracts.

### CODE REVIEW

**Comment by kkowalsky** [comment]
@migurski: think I phrased how a custom extract is made wrong, is the data from the processed planet file or...?

**Comment by rmglennon** [comment]
@sleepylemur says this is correct... I had no idea this is how it worked.

**Comment by rmglennon** [comment]
I moved the steps to paragraphs so numbered steps are only things people have to do.
Changed little things like "click on" and "press" to click.
I don't normally like to use quotation marks around items on the user interface, but don't currently have a better way to identify them (such as 'your custom extracts' page), so let's come up with a writing style for that.

Other companies, like Microsoft, but tags that display bold around UI items. This highlights the UI string in the text, and also makes it easier for localization because the translator can see that it's a UI term that should be updated from a list of specific UI string names.

File: docs/custom-extracts.md [hunk 1]
```diff
@@ -4,18 +4,28 @@ Is the location you're looking for not on the extract list? Mapzen hosts the top
...
+4. After you have the custom extract ready, click Get Extract. You'll be prompted to either sign up or sign in with a Mapzen Developer account. This uses [GitHub](https://www.github.com) authentication (if you don't have a GitHub account, you can sign up for one at: [https://github.com/join](https://github.com/join)).
+
+Custom extracts can take about 30-60 minutes to generate, depending on the size. An email will be sent to the address associated with your GitHub account, but will also be viewable on the Metro Extracts page under 'your custom extracts'. This page also shows you the status of the custom extract and allows you to access previous requests as well.
 ![Pending custom extracts](./images/your_custom_extracts.png)
+
+### Share custom extracts
+
+After your custom extracts have been created, they're viewable on the 'your custom extracts' page. They can be shared with anyone who also has a Mapzen Developer Account by copying the URL and sending it to the other person.
```
Comment (souperneon) [inline docs/custom-extracts.md]: @kkowalsky only individual extracts are shareable via the link, not the entire list.

Comment (kkowalsky) [inline docs/custom-extracts.md]: Will make that more explicit thanks!

**Pull request merged**